ingestion_tools/scripts/data_validation/fixtures/data.py

```diff
@@ -18,24 +18,34 @@
 # ==================================================================================================
 
 
-def get_header(mrcfile: str) -> MrcInterpreter:
-    """Get the mrc file header for a tilt series without downloading the entire file."""
-    fs = FileSystemApi.get_fs_api(mode="s3", force_overwrite=False)
-    with fs.open(mrcfile, "rb") as f:
-        header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
-    return header
-
-
-def get_headers(mrcfiles: List[str]) -> Dict[str, MrcInterpreter]:
+def get_header(mrcfile: str, fs: FileSystemApi) -> MrcInterpreter:
     """Get the mrc file headers for a list of mrc files."""
-    headers = {}
-    for file in mrcfiles:
-        try:
-            headers[file] = get_header(file)
-        except Exception as _:
-            pytest.fail(f"Failed to get header for {file}")
-
-    return headers
+    try:
+        with fs.open(mrcfile, "rb") as f:
+            header = MrcInterpreter(iostream=f, permissive=True, header_only=True)
+        return header
+    except Exception as _:
+        pytest.fail(f"Failed to get header for {mrcfile}")
+
+
+def get_zattrs(zarrfile: str, fs: FileSystemApi) -> Dict:
+    """Get the zattrs for a zarr volume file without downloading the entire file."""
+    with fs.open(zarrfile + "/.zattrs", "r") as f:
+        zattrs = json.load(f)
+    return zattrs
+
+
+def get_zarrays(zarrfile: str, fs: FileSystemApi) -> Dict:
+    """Get the zarray for a zarr volume file without downloading the entire file."""
+    zarrays = {}
+    expected_subfolders = [f"{zarrfile}/{i}" for i in range(3)]
+    actual_subfolders = ["s3://" + folder for folder in fs.glob(zarrfile + "/*/")]
```

Comment:
This is fine, but we can also use the omezarr reader to do the same thing. I think we should start thinking about whether we want to use filesystem checks vs omezarr reader checks more consistently.